### PR TITLE
welding hats play the visor up/down sfx instead of MOD hiss

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
changes the visor up/down sfx for welding hats to be `SFX_VISOR_UP`/`SFX_VISOR_DOWN`
## Why It's Good For The Game
It makes much more sense
## Changelog
:cl: grungussuss
sound: changed the welding hat toggle sfx
/:cl:

### CODE REVIEW

File: code/modules/clothing/spacesuits/plasmamen.dm [hunk 1]
```diff
@@ -139,7 +139,10 @@
 		to_chat(user, span_notice("Your helmet's torch can't pass through your welding visor!"))
 		set_light_on(FALSE)
 		helmet_on = FALSE
-	playsound(src, 'sound/vehicles/mecha/mechmove03.ogg', 50, TRUE) //Visors don't just come from nothing
+	if(up)
+		playsound(src, SFX_VISOR_UP, 50, TRUE)
+	else
+		playsound(src, SFX_VISOR_DOWN, 50, TRUE) //Visors don't just come from nothing
```
Comment (MrMelbert) [inline code/modules/clothing/spacesuits/plasmamen.dm:145]:
```suggestion
		playsound(src, SFX_VISOR_DOWN, 50, TRUE) 
```

**Review by MrMelbert — COMMENTED** [review]
If it were me I'd format them as `playsound(src, up ? SFX_VISOR_UP : SFX_VISOR_DOWN, 50, TRUE)`, but I don't mind that much

File: code/modules/clothing/spacesuits/plasmamen.dm (hunk 1 above)
Comment (Sadboysuss) [reply code/modules/clothing/spacesuits/plasmamen.dm]: [done](https://github.com/tgstation/tgstation/pull/87920/commits/c59ac5da15feee274457af7bfe21462b7c9d0faf)

File: tgstation.dme [hunk 2]
```diff
@@ -244,6 +244,7 @@
 #include "code\__DEFINES\time.dm"
 #include "code\__DEFINES\tools.dm"
 #include "code\__DEFINES\toys.dm"
+#include "code\__DEFINES\tracy.dm"
```
Comment (Ben10Omintrix) [inline tgstation.dme:247]:
```suggestion
```
think something mightve gone wrong

**Review by Ben10Omintrix — CHANGES_REQUESTED** [review]

File: tgstation.dme (hunk 2 above)
Comment (Sadboysuss) [reply tgstation.dme]: merge skew it seems

**Pull request merged**